Maddori.Apple/Maddori.Apple/Global/UIComponent/FeedbackTypeButtonView.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  FeedbackTypeButtonView.swift
+//  Maddori.Apple
+//
+//  Created by 김유나 on 2022/10/19.
+//
+
+import UIKit
+
+import SnapKit
+
+final class FeedbackTypeButtonView: UIButton {
+    private enum FeedbackType {
+        case continueType
+        case stopType
+    }
+    private enum Size {
+        static let buttonPadding: CGFloat = 11
+        static let width: CGFloat = 158
+        static let height: CGFloat = 46
+    }
+    
+    // MARK: - property
+    
+    private let continueShadowView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white100
+        view.layer.shadowOpacity = 0.2
+        view.layer.shadowRadius = 2
+        view.layer.shadowOffset = CGSize(width: 0, height: 0)
+        view.layer.cornerRadius = 10
+        return view
+    }()
+    private let stopShadowView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white100
+        view.layer.shadowOpacity = 0.2
+        view.layer.shadowRadius = 1
+        view.layer.shadowOffset = CGSize(width: 0, height: 0)
+        view.layer.cornerRadius = 10
+        return view
+    }()
+    private lazy var continueButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("Continue", for: .normal)
+        button.setTitleColor(.white100, for: .normal)
+        button.titleLabel?.font = .main
+        button.backgroundColor = .blue200
+        button.clipsToBounds = true
+        button.layer.cornerRadius = 10
+        let action = UIAction { [weak self] _ in
+            self?.touchUpToSelectType(.continueType)
+        }
+        button.addAction(action, for: .touchUpInside)
+        return button
+    }()
+    private lazy var stopButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("Stop", for: .normal)
+        button.setTitleColor(.gray600, for: .normal)
+        button.titleLabel?.font = .main
+        button.backgroundColor = .white100
+        button.clipsToBounds = true
+        button.layer.cornerRadius = 10
+        let action = UIAction { [weak self] _ in
+            self?.touchUpToSelectType(.stopType)
+        }
+        button.addAction(action, for: .touchUpInside)
+        return button
+    }()
+    
+    // MARK: - life cycle
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        render()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func render() {
+        self.addSubview(continueShadowView)
+        continueShadowView.snp.makeConstraints {
+            $0.width.equalTo(Size.width)
+            $0.height.equalTo(Size.height)
+        }
+        continueShadowView.addSubview(continueButton)
```

Comment:
p4;

```suggestion
        }
        
        continueShadowView.addSubview(continueButton)
```
한 줄만.... (굽신굽신)

Reply:
[ecd04df](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/27/commits/ecd04dfe50779c3d571578aa1ea3b9e408b6d7a2)
⭐️ 반영 완료 ⭐️